app/src/sharedTest/java/org/oppia/app/screenshottesting/OnboardingActivityScreenshotTest.kt

```diff
@@ -0,0 +1,40 @@
+package org.oppia.app.screenshottesting
+
+import android.os.Environment
+import androidx.test.core.app.ActivityScenario.launch
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.onboarding.OnboardingActivity
+import org.oppia.testing.screenshots.OUTPUT_FOLDER_NAME
+import org.oppia.testing.screenshots.ScreenshotManager
+import java.io.File
+
+@RunWith(AndroidJUnit4::class)
+class OnboardingActivityScreenshotTest {
+
+  private val screenshotManager = ScreenshotManager()
+
+  @Before
+  fun setup() {
+    createOutputFolder()
+  }
+
+  @Test
+  fun testOnboardingActivity_takeScreenshot() {
+    launch(OnboardingActivity::class.java).use {
+      it.onActivity { activity ->
+        screenshotManager.takeScreenshot(activity)
```

Comment:
Just a thought, can we somehow check if the file is saved/stored at correct location.
Currently we are just running this function but what if this function is incorrect and fails, then also the test will always pass.

I know this is not part of this task but somehow if we can check that file is actually getting created, that would be great.

Reply:
Done